website/docs/ops/migrate-to-mysql.md

```
@@ -0,0 +1,326 @@
+---
+id: migrate-to-mysql
+title: 从 SQLite / PostgreSQL 迁移到 MySQL
+sidebar_position: 3
+---
+
+# 从 SQLite / PostgreSQL 迁移到 MySQL {#migrate-to-mysql}
+
+本文档适用于已使用 SQLite 或 PostgreSQL 部署 ArcReel、希望切换到 **MySQL 8.0+** 的场景。
+如果是全新部署，可直接跳到 [3. 环境变量配置](#configure-env) 并参考 [部署与运维](./deployment.md#mysql-deployment)。
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)